clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/api/KubernetesClientApiAdapter.groovy

```diff
@@ -361,5 +361,15 @@ class KubernetesClientApiAdapter {
       }
     }
   }
+
+  String getControlleType(String name, String namespace) {
+    exceptionWrapper("getControlleType", "Get controller type ${name}", namespace) {
+      def deployedControllerSet = getStatefulSet(name, namespace)
```

Comment:
What if it's a replica set, or a deployment, or a job?

Reply:
the call will ignore these controller type and continue.